mvtprovider/postgis/doc.go

```diff
@@ -1,6 +1,5 @@
 // Package postgis is a placeholder for the postgis database.
 // The postgis provider is provided by the provider.Postgis provider
 // please consult that for information on how to use it.
-// please note that the provider will be called mvt_postgis in the
-// map layers
+// please note that the provider type will be mvt_postgis
 package postgis
```

Comment:
should we maybe just name this package `mvt_postgis`?

Reply:
Generally you don't want "_" in the name of pacakges. Also, it's already in the mvtprovider package, and it seems redundant to have the mvt there again.